### PR TITLE
Change function format to fstring on asv/graph.py

### PR DESCRIPTION
**Pull request by Daniet**
xref #1175 #1077
Change f-strings instead .format() on asv/graph.py

### CODE REVIEW

**Review by dorothykiz1 — APPROVED** [review]
@Daniet thank you for working, great clean up

**Review by LucyJimenez — APPROVED** [review]
Thank you, @Daniet  LGTM 🚀

File: asv/graph.py [hunk 1]
```diff
@@ -118,11 +118,11 @@ def get_file_path(cls, params, benchmark_name):
...
         for key, val in l:
             if val is None:
-                part = '{0}-null'.format(key)
+                part = f'{key}-null'
             elif val:
-                part = '{0}-{1}'.format(key, val)
+                part = f'{key}-{val}'
             else:
-                part = '{0}'.format(key)
+                part = f'{key}'
```
Comment (datapythonista) [inline asv/graph.py:125]: Probably better to simply use `str(key)` instead

**Comment by datapythonista** [comment]
Thanks @Daniet

**Pull request merged**